README.md

```diff
@@ -2,50 +2,89 @@
 
 **Absolutely DO NOT use with mainnet funds!!! No privacy - address reuse and other problems; no security review!**
 
-This code is intended for demonstration only.
+TL;DR - run `cargo help` and read `main.rs`.
+
+This code is intended for demonstration purposes only.
 If you read it it should give you a better picture of [`rust-bitcoin`](https://docs.rs/bitcoin) and
 associated crates.
-Note that this is just basic introduction, not a comprehensive developer guide.
-It doesn't teach about security or other important practices - it'd require whole book, maybe even multiple books!
+Note that this is just basic a introduction, not a comprehensive developer guide.
+It doesn't teach about security or other important practices - which would require a whole book, maybe even multiple books!
 
-The code has regtest hard-coded. This would be normally a bad practice if this wasn't a demo.
+The code has regtest hard-coded, hard-coding the network should not be done in production code.
 
 ## Tutorial
 
 The `master` branch contains only the skeleton which you can use to implement the wallet yourself.
 Implementing your own wallet will give you much better insight into Bitcoin and its Rust library.
-The skeleton provides configuration and database support.
+The skeleton provides configuration and database support plus some minimal helper functions.
+
+
+### Overview
+
+0. Install/configure bitcoind regtest node (see below).
+1. Configure `pico-bitcoin-wallet` (see below).
+2. Read the code in `lib.rs`
+3. Implement the `address` command.
+4. Implement the `scan` command.
+5. Implement the `send` command.
+
+
+Solutions are provide per-step as git branches.
```

Comment:
provided

Reply:
Looks like you haven't pushed them. Maybe better to wait till closer to the conf anyways so we don't have to go back and change any of the branches.

Reply:
I meant typo. But I didn't write them yet, so maybe leave it out for now?